conans/test/unittests/client/generators/pkg_config_test.py

```diff
@@ -16,6 +16,7 @@ def variables_setup_test(self):
         conanfile.initialize(Settings({}), EnvValues())
         ref = ConanFileReference.loads("MyPkg/0.1@lasote/stables")
         cpp_info = CppInfo("dummy_root_folder1")
+        cpp_info.name = ref.name
```

Comment:
Shouldn't you try to change the name and test something?

Reply:
Sure, done!